### PR TITLE
[chore] Re-record JSON responses for docs

### PR DESCRIPTION
**Pull request by nwithan8**
- Most JSON responses have not changed outside of timestamps. 
- Some have extra data entries (previously one or two records, now three or four).
- Some deprecated functionality (carbon offset, batch create+buy) have been removed.
- Added Python tool used to generate these response snippets + improved documentation and Makefile steps

### CODE REVIEW

**Review by Justintime50 — DISMISSED** [review]
I'd like these regenerated using the correct user ID for the responses please, could be some weird artifacts mixed in if not (see internal diff for more details on that)

**Comment by nwithan8** [comment]
> I'd like these regenerated using the correct user ID for the responses please, could be some weird artifacts mixed in if not (see internal diff for more details on that)

I have addressed your feedback and re-recorded all the cassettes with the proper API keys, did result in a lot less diffs.

I've also added the Python tool we use to generate these snippets into the `tools` directory and improved its documentation so it will be easier next time around to carry out this process.

File: official/docs/responses/addresses/addresses-create-and-verify.json [hunk 1]
```diff
@@ -22,26 +22,12 @@
     "verifications": {
       "zip4": {
         "success": true,
-        "errors": [
```
Comment (Justintime50) [inline official/docs/responses/addresses/addresses-create-and-verify.json:25]: thought: we probably want to change the address here to one that will error. The entire point of this example was to show this in the response

File: official/docs/responses/addresses/addresses-verify.json [hunk 2]
```diff
@@ -22,26 +22,12 @@
     "verifications": {
       "zip4": {
         "success": true,
-        "errors": [
```
Comment (Justintime50) [inline official/docs/responses/addresses/addresses-verify.json:25]: thought: we probably want to change the address here to one that will error. The entire point of this example was to show this in the response

File: official/tools/build_doc_json_responses/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+# Run the following to save HTTP recordings
+EASYPOST_TEST_API_KEY=putrealkeyhere EASYPOST_PROD_API_KEY=putrealkeyhere PARTNER_USER_PROD_API_KEY=putrealkeyhere REFERRAL_CUSTOMER_PROD_API_KEY=putrealkeyhere make generate
+
+# Run the same command again to save the response snippets
+EASYPOST_TEST_API_KEY=putrealkeyhere EASYPOST_PROD_API_KEY=putrealkeyhere PARTNER_USER_PROD_API_KEY=putrealkeyhere REFERRAL_CUSTOMER_PROD_API_KEY=putrealkeyhere make generate
+```
+
+**NOTE**: Some static data used by this tool, such as the USPS carrier account ID, is associated with the EasyPost Docs Tooling User. You must use that user's API keys to generate these snippets.
+
+Generate response snippets are stored in the `responses` directory. Copy this entire directory to `official/docs/responses` once complete.
```
Comment (Justintime50) [inline official/tools/build_doc_json_responses/README.md:31]:
```suggestion
Generated response snippets are stored in the `responses` directory. Copy this entire directory to `official/docs/responses` once complete.
```

File: official/tools/build_doc_json_responses/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Linting
+
+The `make lint` and `make lint-fix` commands in the Makefile will lint the Python code in this tool, but not the response snippets.
+
+After copying the response snippets to `official/docs/responses`, you can lint them with the `make format-node` command in the root Makefile, which will lint and format all JSON files, including the response snippets, in the `official/docs` directory.
+
+## Troubleshooting
+
+### A bunch of "F"s
```
Comment (Justintime50) [inline official/tools/build_doc_json_responses/README.md:43]: thought: this is a redefinition of another paragraph above, seems odd to repeat it said slightly differently. Can we consolidate them?

**Review by jchen293 — APPROVED** [review]

**Review by jchen293 — APPROVED** [review]

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**